docroot/profiles/custom/cgov_site/themes/custom/ncids_trans/templates/content/mini_landing/node--cgov-mini-landing--ncids-default--full.html.twig

```diff
@@ -1,4 +1,7 @@
 <!-- Begin MAIN CONTENT AREA -->
 <main class="main-content type {{ page_type }}" id="main-content">
+  <div class="usa-section usa-section--cgdp-no-bottom" data-eddl-landing-row>
+    <h1 class="nci-heading-h1">{{ drupal_token('current-page:title')}}</h1>
```

Comment:
You should be able to just reference the node's title here, `<h1>{{ node.label }}</h1>`, and skip drupal_token. drupal_token should only be used when you can't get the data through a more direct (and more performant) way.